pyramid/predicates.py

```diff
@@ -130,16 +130,26 @@ def __call__(self, context, request):
         return self.val.match(val) is not None
 
 class AcceptPredicate(object):
-    def __init__(self, val, config):
-        self.val = val
+    _is_using_deprecated_ranges = False
+
+    def __init__(self, values, config):
+        if not is_nonstr_iter(values):
+            values = (values,)
+        # deprecated media ranges were only supported in versions of the
+        # predicate that didn't support lists, so check it here
+        if len(values) == 1 and '*' in values[0]:
+            self._is_using_deprecated_ranges = True
+        self.values = values
 
     def text(self):
-        return 'accept = %s' % (self.val,)
+        return 'accept = %s' % (', '.join(self.values),)
 
     phash = text
 
     def __call__(self, context, request):
-        return self.val in request.accept
+        if self._is_using_deprecated_ranges:
```

Comment:
Knowing that this will warn a `DeprecatedWarning`, should we filter that here knowing that we already told the developer during config time?